README.md

```diff
@@ -1,16 +1,27 @@
-[![Build Status](https://api.travis-ci.org/47deg/scala-exercises.svg?branch=version-2)](https://travis-ci.org/47deg/scala-exercises?branch=version-2) [![Stories in Ready](https://badge.waffle.io/47deg/scala-exercises.png?label=ready&title=Ready)](https://waffle.io/47deg/scala-exercises)
+[![Stories in Ready](https://badge.waffle.io/47deg/scala-exercises.png?label=ready&title=Ready)](https://waffle.io/47deg/scala-exercises)
+[![Build status](https://badges.gitter.im/Join%20Chat.svg)](https://travis-ci.org/47deg/scala-exercises.svg?branch=master)
 
-[![Join the chat at https://gitter.im/47deg/scala-exercises](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/47deg/scala-exercises?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+#Scala Exercises
 
-# Scala Exercises
+------------------------
 
-This is second iteration of Scala Exercises. Version 1 is available by going to the
-[master](https://github.com/47deg/scala-exercises/tree/master) branch.
+## How it works
+
+"Scala Exercises" brings exercises for the Stdlib, Cats, Shapeless and many other great libraries for Scala. Offering hundreds of solvable exercises organized into several categories covering the basics of the Scala language and it's most important libraries.
```

Comment:
What about ending the first phrase like this: "brings exercises [...] other great libraries for Scala **to your browser**"